### ISSUE TITLE
[ch_seco_sanctions] Improve "other-information" parser

### ISSUE THREAD

**Issue by pudo** [opensanctions/opensanctions#1182]
We should introduce a lookup-based parser for the detailed information provided in the `<other-information>` section of the data

**Comment by jbothma** [comment]
I think that was mainly because of the number of times I iterated towards names and most other values being a list, because somewhere there were multiple alternatives. A quick check only shows one strong candidate for it.

And yes both values are used but it's passed to make_id as one list rather than as a list of args. I'll fix.

<img width="1192" height="203" alt="Image" src="https://github.com/user-attachments/assets/7a11a32b-753e-43b8-a454-f3b9e5d283b3" />

**Comment by goose-life** [comment]
@pudo do you feel like the extraction quality has improved enough here to call this done? 
(all that's outstanding otherwise is going through the ~3k reviews)

**Comment by pudo** [comment]
Yeah to me the issue here is just the quiet reminder that we don't have any way to do 3k reviews. One thing that could still enable this to become more feasible: drop relationship extraction. The property extraction is usually correct, but almost all extracted links need manual fixes.
